docs/testing.md

```diff
@@ -163,7 +144,7 @@ A number of command-line arguments can be set to test particular configurations:
  - `--suite`: Choose a particular suite or suites to run.
    For example, `gulp test:acceptance --suite=content` or `gulp test:acceptance --suite=content,functional`.
  - `--specs`: Choose a particular spec or specs to run.
-   For example, `gulp test:acceptance --specs=contact-us.js`, `gulp test:acceptance --specs=contact-us.js,about-us.js`, or `gulp test:acceptance --specs=foo*.js`. If `--suite` is specified, this argument will be ignored. If neither `--suite` nor `--specs` are specified, all specs will be run.
+   For example, `gulp test:acceptance --specs=header.feature`, `gulp test:acceptance --specs=header.feature, pagination.feature`, or `gulp test:acceptance --specs=filterable*.feature`. If `--suite` is specified, this argument will be ignored. If neither `--suite` nor `--specs` are specified, all specs will be run.
```

Comment:
Will `gulp test:acceptance --specs=header.feature, pagination.feature` work with the space after the comma in the `--specs` flag?

Reply:
lol, that is fragile

Reply:
Yeah, just tested and the space is a problem.